openproblems/data/allen_brain_atlas.py

```diff
@@ -0,0 +1,47 @@
+from . import utils
+
+import numpy as np
+import os
+import scanpy as sc
+import scprep
+import tempfile
+
+URL = "https://figshare.com/ndownloader/files/36509385"
```

Comment:
Can you please add a comment linking to the figshare.com/articles/... URL that describes this figshare dataset?